documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -0,0 +1,394 @@
+// This module is included in the following files:
+//
+// master.adoc
+
+[id='con-common-configuration-properties-{context}']
+== Common configuration properties
+
+Common configuration properties apply to more than one resource.
+
+[id='con-common-configuration-replicas-{context}']
+.`replicas`
+
+Use the `replicas` property to configure replicas.
+
+The type of replication depends on the resource.
+
+* `KafkaTopic` uses a replication factor to configure the number of replicas of each partition within a Kafka cluster.
+* Kafka components use replicas to configure the number of pods in a deployment to provide better availability and scalability.
+
+*MirrorMaker*
+
+When running Kafka MirrorMaker on Kubernetes it is not absolutely necessary to run multiple replicas of the Kafka MirrorMaker for high availability.
+When the node where the Kafka MirrorMaker has deployed crashes, Kubernetes will automatically reschedule the Kafka MirrorMaker pod to a different node.
+However, running Kafka MirrorMaker with multiple replicas can provide faster failover times as the other nodes will be up and running.
```

Comment:
This applies to also for example Kafka Connect. But it seems a bit weird to mention it explicitely in section called _Common configuration properties_.